fog/ingest/server/src/ingest_service.rs

```diff
@@ -174,6 +174,19 @@ where
 
         Ok(response)
     }
+
+    /// Retrieves a private key from a remote encalve and then sets it as the
+    /// current enclave's private key.
+    pub fn sync_keys_from_remote_impl(
+        &mut self,
+        request: SyncKeysFromRemoteRequest,
+        logger: &Logger,
+    ) -> Result<IngestSummary, RpcStatus> {
+        let peer_uri = IngestPeerUri::from_str(request.get_peer_uri()).unwrap();
```

Comment:
This should not use `unwrap` - using `unwrap` here means that an invalid URL will crash the server.
Instead, if should `map_err` into `RpcStatus` and return that in case of invalid URL.

Reply:
Thanks for pointing that out- fixed.